docs/cni/kubernetes/AWS.md

```diff
@@ -123,7 +123,7 @@ aws ec2 create-security-group --group-name MySG \
   --description MySecurityGroup --vpc-id $VPC_ID
```

Comment:
Hmm.  When you say that the keyfile "should include the --BEGIN--- and ----END--- and everything in between", you might like to say "must ONLY include".